packages/cli/src/command.ts

```diff
@@ -48,6 +47,34 @@ export const SYNC_OPTION_FLAG = Flags.integer({
   },
 })
 
+const readPipe: () => Promise<string | undefined> = () => {
```

Comment:
Could use https://github.com/sindresorhus/get-stdin maybe rather than a custom implementation, unless the logic differs?

Reply:
My function adds a timeout to make sure the stdin stream isn't left hanging, even if there's no `end` signal (which for some reason happens when running the commands in tests via `execa(...)`. Please, have a look at my other comment below and let me know what you think :).